src/components/PlantDetailLayout/PlantDetailLayout.tsx

```diff
@@ -1,26 +1,39 @@
-import { SafeAreaView } from 'react-native-safe-area-context';
+import { SafeAreaView, useSafeAreaInsets } from 'react-native-safe-area-context';
 import { useTranslation } from 'react-i18next';
 import { useStyles } from 'react-native-unistyles';
 import { View } from 'react-native';
 import { useLocalSearchParams } from 'expo-router';
-import React from 'react';
+import React, { useState } from 'react';
 
 import { PlantDetailTab } from '@src/components/PlantDetailTab/PlantDetailTab';
 import { PlantPhotoSwiper } from '@src/components/PlantPhotoSwiper/PlantPhotoSwiper';
 
 import { styleSheet } from './PlantDetailLayout.style';
 import { PlantDetailTabBtns } from '../PlantDetailTabBtns/PlantDetailTabBtns';
+import { CustomStatusBar } from '../CustomStatusBar/CustomStatusBar';
+import { STATUS_BAR_TEXT_COLOR } from '../CustomStatusBar/CustomStatusBar.constants';
 
 export const PlantDetailLayout = () => {
 	const { t } = useTranslation();
 	const { styles } = useStyles(styleSheet);
 	const { uuid } = useLocalSearchParams<{ uuid?: string }>();
+	const [hasPlanPhoto, setHasPlanPhoto] = useState<boolean>(false);
+
+	const setIsPhotoState = () => {
+		setHasPlanPhoto(true);
+	};
 
 	const plantUuid = uuid ?? '';
+
+	const statusBarHeight = useSafeAreaInsets().top;
+	const statusBarHeightToAdd = hasPlanPhoto ? 0 : statusBarHeight;
```

Comment:
Why do we need this?

Reply:
> Why do we need this?

We are considering the idea of not displaying the component at all if we don't have the necessary data. Therefore, the component located at the top should have additional padding for the status bar. We handle this by checking if the necessary information exists, and if it does, we add padding only to the component that appears at the top of the page